package/libs/openssl/Makefile

```diff
@@ -14,9 +14,6 @@ PKG_VERSION:=$(PKG_BASE)$(PKG_BUGFIX)
 PKG_RELEASE:=1
 PKG_USE_MIPS16:=0
 
-PKG_BUILD_PARALLEL:=0
-
-
```

Comment:
and this one

Reply:
Can't reproduce any build issues
It does however revert to -j1 in some parts of the build, I don't think it's worth spending much time on as we'll move to 1.1.1 soon.